news/2018-03-06-new-featured-artist-trial-and-error.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+title: "New Featured Artist: Trial & Error"
+date: 2018-03-06 11:00:00 +0000
+---
+
+The search for musical talent never ends, and we're proud to announce the next addition to osu!'s Featured Artist lineup - **Trial & Error**!
+
+[![](https://assets.ppy.sh/artists/20/header.jpg)](https://osu.ppy.sh/beatmaps/artists/20)
+
+The mind behind countless Japanese and anime-themed tracks, **Trial & Error**'s work has been featured in popular projects like the cult classic visual novel *Stay! Stay! Democratic People's Republic of Korea* and *Panzermadels: Tank Dating Simulator*. 
+
+A multi-discipline master of music production, **Trial & Error**'s work crosses a variety of sounds and genres, ranging from the quintessential anime hits that have become massively prominent in osu!, to soulful ballads and grungy rock hits, supported with a variety of vocalists both real and virtual.
+
+But don't take our word for it, have a listen to a small sample of what's available and see for yourself!
+
+### Yoru wo Kakeru feat. ming-zi
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/20/previews/383.mp3" type="audio/mpeg">
+</audio>
+
+> An airy vocal walk featuring the talent of ming-zi, Yoru wo Kakeru levels traditional instrumentation together with sweeping lyrics. Nothing particularly new, but an excellent take on a more standard musical approach.
+
+### DAI★TAN SENSATION feat. Nanahira, Mii, Nakuru Aitsuki
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/20/previews/381.mp3" type="audio/mpeg">
+</audio>
+
+> Uplifting and with all the sugar and sweetness expected from an anime-styled track, DAI★TAN SENSATION is a playful romp among the genre, with the added bonus of the vocals from a few prominent singers among the Japanese music scene!
+
+### Hoshizora Touhikou feat. Hatsune Miku
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/20/previews/372.mp3" type="audio/mpeg">
+</audio>
+
+> A heartfelt ballad rendered in the voice of the Vocaloid Hatsune Miku, Hoshizora Touhikou is a slow and paced walk throughout an sombre starscape, tied off with gentle, leading percussion towards the later parts of the track.
+
+### Tokoyami no Keiyaku KEGARETA-SHOUJO feat. GUMI
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/20/previews/377.mp3" type="audio/mpeg">
+</audio>
+
+> Beset with a backing choir, the voice of the Vocaloid GUMI reigns supreme in this fast-paced, rock-themed piece.
+> 
+
+### Houkai no Hate No Ishi feat. GUMI
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/20/previews/373.mp3" type="audio/mpeg">
+</audio>
+
+> Slower paced and more pensive, Houkai no Hate No Ishi is a faintly ethereal take on a classic rock sound, set to the voice of the Vocaloid GUMI.
+
+------
+
+But that's not all, there's a few more tracks on [**Trial & Error**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/20) for you to check out if you're interested. 
```

Comment:
listing -> page

listing lists all artists i think

interested -> interested in?